src/transformers/pipelines/audio_classification.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import subprocess
-from typing import Union
+from typing import Any, Dict, List, Union
```

Comment:
From readme I see transformers supports py 3.9+. According to [py doc], there is no need to use typing.Dict and List.

[py doc]: https://docs.python.org/3/library/typing.html#typing.Dict

Reply:
Thanks! Will replace 🤗